resources/dicts/patrols/plains/training/any.json

```diff
@@ -517,18 +538,26 @@
                     "vengeful",
                     "bossy"
                 ],
+                "can_have_stat": ["r_c"],
                 "relationships": [
                     {
-                        "cats_to": ["s_c", "p_l", "p_l", "s_c"],
-                        "cats_from": ["patrol", "patrol", "r_c", "r_c"],
+                        "cats_to": ["s_c"],
+                        "cats_from": ["o_c1"],
```

Comment:
o_c1?

Reply:
it was listed as a tag in the new patrol bible, but my bad if it's only for events or something (it wasn't specified); I do not see any way to make this outcome work other than perhaps replacing o_c1 with p_l, because r_c is already s_c here (originally this prompt used r_c as the cat insulting the other, but r_c already was the cat who got stuck). That is my mistake for not looking into it further, though I did ponder over this outcome specifically due to this issue, but perhaps I should've thought over it longer

Reply:
looking back at the bible it does actually say it's referring to another cat on the patrol

Reply:
Tested, it seems o_c1 works correctly in patrol text
![image](https://github.com/ClanGenOfficial/clangen/assets/128231119/3fc63e57-791c-42cf-adfd-a6e1ae14c3ec)
But potentially doesn't work in relationships?
![image](https://github.com/ClanGenOfficial/clangen/assets/128231119/c7cbf83d-be60-4713-bcd6-0076bdce7cbb)

Reply:
Changed to p_l, but should be looked into.